boards/arm/stm32/stm32f4discovery/src/stm32_bringup.c

```diff
@@ -72,6 +72,10 @@
 #include "stm32_bmp180.h"
 #endif
 
+#ifdef CONFIG_SENSORS_MS5611
+#include "stm32_ms5611.h"
```

Comment:
```suggestion
#  include "stm32_ms5611.h"
```

Reply:
Not need, please note that all other include doesn't have this space:
```
#ifdef CONFIG_SENSORS_BMP180
#include "stm32_bmp180.h"
#endif

#ifdef CONFIG_SENSORS_MS5611
#include "stm32_ms5611.h"
#endif

#ifdef CONFIG_SENSORS_MAX6675
#include "stm32_max6675.h"
#endif

#ifdef CONFIG_INPUT_NUNCHUCK
#include "stm32_nunchuck.h"
#endif

#ifdef CONFIG_SENSORS_ZEROCROSS
#include "stm32_zerocross.h"
#endif
```

Reply:
This is something that is not well-defined or enforced by the coding style. Case it became an enforcement it need to be detected by the nxstyle to avoid people forget to follow it.